src/en/guarantees.md

```diff
@@ -38,3 +38,33 @@ However, the language does not prevent
 
 * resource leaks (memory, IO, ...),
 * numeric overflows.
+
+## LLVM Tier guarantees
```

Comment:
I would remove LLVM from the title.

It may come a time where Rustc switchs backend for some platform (in particular for wasm, or for debug builds).